File: API/TECAirDbAPI/Models/Flight.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace TECAirDbAPI.Models
{
    public partial class Flight
    {

        public string Flightid { get; set; }
        public int? Bagquantity { get; set; }
        public int? Userquantity { get; set; }
        public int Gate { get; set; }
        public DateTime Departure { get; set; }
        public DateTime Arrival { get; set; }
        public string Origin { get; set; }
        public string Destination { get; set; }
        public string Stops { get; set; }
        public string Status { get; set; }
        public int? Price { get; set; }
        public int? Discount { get; set; }
        public string Planeid { get; set; }
        public int? Workerid { get; set; }
    }
}

File: Reservaciones/src/app/Pages/models/flight.ts
/**
 * @description: Interface of flights
 */
export interface Flight {
  origin: string,
  destination: string,
  bagquantity: number,
  userquantity: number,
  flightid: string,
  departure: string,
  arrival:string
  price:number,
  stops:[],
  gate: number,
  status: string,
  discount: number,
  planeid: string,
  workerid: number
}
